### ISSUE TITLE
New translatable strings added into to "team.inc" - tested by Rytis on PrimeGrid.

### ISSUE THREAD

**Issue by romw** [BOINC/boinc-ng#638]
**Reported by Simek on 17 Mar 38378725 21:20 UTC**
I made "team.inc" file translatable and I add those new translations strings to "en.po" file. Two patches for those files included. 

Changes tested by Rytis on PrimeGrid - thanks so much!

**Comment by romw** [comment]
**Commented by Nicolas on 13 May 38378840 01:46 UTC**
That is a '''mess''' to translate. How can somebody translate "without" or "You may use" if he doesn't actually look at where the strings are used?

Given this:
```tra("You may use")." <a href=\"html.php\" target=\"_new\">".tra("limited HTML tags")."```

What if in some language the only way to say that is with "HTML tags" before "you"? Not every language has the same grammatical order. Sometimes the only way to say it is like "limited HTML tags may be used", and that's impossible with the current patch.

**Comment by romw** [comment]
**Commented by davea on 29 Aug 38379290 16:00 UTC**
Strings should generally be at the level of sentence rather than phrase; e.g.
"You may use %1limited HTML tags%2."

Nicolas, can you make these changes in team.inc?

**Comment by romw** [comment]
**Commented by Nicolas on 26 Mar 38381995 17:46 UTC**
Erm, are you sure this was tested in !PrimeGrid completely? I see things that couldn't possibly work, like commas instead of concatenation operators in some places. It would make "Don't use HTML tags." go into the '''second column''' instead of '''below''' "Team name, text version".

**Comment by romw** [comment]
**Commented by Nicolas on 9 Jul 38382011 19:33 UTC**
Heh nice, I found a few XSS bugs too.

What a mess.

**Comment by romw** [comment]
**Commented by Simek on 22 Jan 38383935 16:26 UTC**
```
Sure; send me the file and I'll try it on PrimeGrid.
```
----
```
Line 19:

row2(tra("Key words") "<br/>
should be
row2(tra("Key words")."<br/>
(notice the missing dot)

Other than that, I haven't spotted any errors.
```
----
Those replies I got for Rytis. Sorry for this mess. I'll try to fix it.

**Comment by romw** [comment]
**Commented by Nicolas on 30 Apr 38384469 13:46 UTC**
I'm working on those files now (It needs somewhat deep changes to get rid of the security bugs). You may want to wait with the translations...

**Comment by romw** [comment]
**Commented by Simek on 5 Mar 38384492 11:33 UTC**
Ok, great. It's no problem, I'll wait untill you finish it...

**Comment by romw** [comment]
**Commented by Nicolas on 22 Oct 38394802 02:40 UTC**
XSS bug is fixed - wasn't as hard as it seemed.

**Comment by romw** [comment]
**Commented by davea on 31 Jan 38394929 19:06 UTC**
For some reason the patch didn't work; see below
Please post new team.inc.patch relative to trunk.

patch -p0 < temp
patching file team.inc
Hunk #1 FAILED at 12.
Hunk #2 FAILED at 37.
Hunk #3 succeeded at 60 (offset 1 line).
Hunk #4 FAILED at 70.
Hunk #6 FAILED at 134.
Hunk #7 FAILED at 160.
Hunk #8 succeeded at 204 (offset 1 line).
Hunk #9 FAILED at 223.
Hunk #11 succeeded at 248 (offset 1 line).
Hunk #13 succeeded at 309 (offset 1 line).
Hunk #15 succeeded at 364 (offset 1 line).
6 out of 15 hunks FAILED -- saving rejects to file team.inc.rej

**Comment by romw** [comment]
**Commented by Rytis on 1 Oct 38593923 16:00 UTC**
(In [15747]) user web: make team.inc translatable, and update en.po using the new script (fix #661)
